tests/Aspire.Hosting.Azure.Tests/AzureAppServiceTests.cs

```diff
@@ -144,11 +144,37 @@ await Verify(manifest.ToString(), "json")
               .UseHelixAwareDirectory();
     }
 
+    [Fact]
+    public async Task AzureAppServiceDoesNotSupportBaitAndSwitchResources()
```

Comment:
Should you rename this test now that its fixed?

Reply:
This is app service. When I add support for containers then this will be relevant.